src/rdo.rs

```diff
@@ -333,13 +398,22 @@ fn compute_tx_distortion<T: Pixel>(
   luma_only: bool,
 ) -> u64 {
   assert!(fi.config.tune == Tune::Psnr);
+  let area = Area::BlockStartingAt { bo: tile_bo.0 };
+  let input_region = ts.input_tile.planes[0].subregion(area);
+  let rec_region = ts.rec.planes[0].subregion(area);
   let mut distortion = if skip {
     sse_wxh(
-      &ts.input_tile.planes[0]
-        .subregion(Area::BlockStartingAt { bo: tile_bo.0 }),
-      &ts.rec.planes[0].subregion(Area::BlockStartingAt { bo: tile_bo.0 }),
+      &input_region,
+      &rec_region,
       bsize.width(),
       bsize.height(),
+      |bias_area, bsize| {
+        compute_distortion_bias(
+          fi,
+          input_region.subregion(bias_area).frame_block_offset(),
+          bsize,
+        )
+      },
```

Comment:
This biases skip blocks, but for non-skip blocks we just use tx_dist. You'll still need to bias tx_dist with the mean.